index.bs

```diff
@@ -894,6 +912,65 @@ A {{Point2D}} represents a location in a two dimensional space. The origin of co
   </pre>
 </div>
 
+## Update camera focus distance and {{takePhoto()}} ## {#example4}
+
```

Comment:
I'd love to have a codepen/jsfiddle/anything (a la [1]) like the other examples, so casual visitors could just click through and see this in action. 


[1] https://codepen.io/miguelao/pen/mAzXpN/left?editors=1010